app/Http/Controllers/EmployeesController.php

```diff
@@ -188,6 +188,15 @@ public function random(Request $request) // möglichst zufällig jemanden auswä
             ->where('still_working', true)
             ->get();
 
+        $employeesAvailable = count(Employee::where('is_available', true)
+                                            ->where('still_working', true)
+                                            ->get());
+        if ($employeesAvailable <= 1)
+        {
+            return redirect('/')->withErrors(['no_employees' => 'There are not enough employees to carry out the generation']);
```

Comment:
in this case the reset of employees never happens right? it should reset if everyone has presented and cooked